.kuzzlerc.sample

```diff
@@ -283,7 +283,9 @@
       "aliases": ["broker"],
       "host": "localhost",
       "port": 7911,
-      "retryInterval": 1000
+      "retryInterval": 1000,
+      "pingTimeout": 50,
```

Comment:
These two new options must be documented in the sample file, using the comments immediately above the `internalBroker` and `proxyBroker` configuration sections.